src/test/test_hs_service.c

```diff
@@ -2285,9 +2285,6 @@ test_export_client_circuit_id(void *arg)
   cp1 = buf_get_contents(conn->outbuf, &sz);
   tt_str_op(cp1, OP_EQ,
             "PROXY TCP6 fc00::ffff:ffff:ffff:ffff:ffff:ffff ::1 65535 42\r\n");
-  tor_free(cp1);
-  extend_info_free(chosen_exit);
-  or_circ->build_state->chosen_exit = NULL;
```

Comment:
It's actually safe to call tor_free() and extend_info_free() here, because they set their arguments to NULL.

We sometimes write code with *_free() at the end of every test, so that the next person who adds a test doesn't have to remember to add the extra *_free() calls?

Reply:
Fixed in `06a16fc`.